pages/genre/genre.pixy

```diff
@@ -1,5 +1,6 @@
-component Genre(genre string, animes []*arn.Anime, user *arn.User)
+component Genre(genre string, animes []*arn.Anime, user *arn.User, userScore string)
 	h1(title=fmt.Sprint(len(animes)) + " anime")= strings.Title(genre)
+	h2()= userScore
```

Comment:
You can write `h2= userScore` here, no brackets needed.

If you want to format a float, you can use `fmt.Sprintf` with something like `%.1f` where 1 is the float precision.